src/main/scala/com/typesafe/sbt/packager/debian/DebianArtifactoryDeployPlugin.scala

```diff
@@ -0,0 +1,85 @@
+package com.typesafe.sbt
+package packager
+package debian
+
+import sbt._
+import sbt.Keys._
+import SbtNativePackager._
+import packager.Keys._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/1938988902/issues/source?bid=2040925&fileBranchId=3104082#l8)